src/app/components/DynamicContentStyles.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { query } from '../navigation/selectors';
+
+// tslint:disable-next-line: variable-name
+export const WithStyles = styled.div`
+  ${(props: { styles: string }) => props.styles}
```

Comment:
can you rearrange this so that it doesn't add yet another arbitrary div to the dom structure?